saleor/graphql/order/utils.py

```diff
@@ -97,10 +97,31 @@ def validate_order_lines(order, country):
                 raise ValidationError({"lines": errors})
 
 
+def validate_variants_is_available(order):
```

Comment:
In the original PR we have function `validate_variants_available_in_channel` which does similar validation but for checkouts. Maybe we could unify them and have one?